apps/app/src/views/routes/Patients.tsx

```diff
@@ -17,23 +17,32 @@ import { FiEdit, FiEye, FiMoreVertical, FiShoppingCart } from 'react-icons/fi';
 import { TbPrescription } from 'react-icons/tb';
 
 import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { gql, useQuery } from '@apollo/client';
 import { useDebounce } from 'use-debounce';
-import { usePhoton } from '@photonhealth/react';
 import { Page } from '../components/Page';
 import { TablePage } from '../components/TablePage';
 import PatientView from '../components/PatientView';
 import ContactView from '../components/ContactView';
 import { Patient } from 'packages/sdk/dist/types';
 
-const dobToAge = require('dob-to-age');
-
-declare global {
-  namespace JSX {
-    interface IntrinsicElements {
-      'photon-patient-dialog': unknown;
+const GET_PATIENTS = gql`
+  query GetPatients($after: ID, $name: String, $first: Int) {
+    patients(after: $after, first: $first, filter: { name: $name }) {
+      id
+      externalId
+      phone
+      email
+      sex
+      dateOfBirth
+      name {
+        full
+      }
     }
   }
-}
+`;
+
+const dobToAge = require('dob-to-age');
```

Comment:
We can't do this via an import? (And probably should go at the top with other imports)